.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         uses: actions/setup-java@v2
         with:
           distribution: 'adopt'
-          java-version: '8'
+          java-version: '11'
```

Comment:
this change depends on https://github.com/getsentry/sentry-android-gradle-plugin/issues/179

Reply:
https://github.com/getsentry/sentry-android-gradle-plugin/pull/181/commits/86a4fd1c1e75b48536c6313403461fc8767ccec2